lib/styles/components/_badge-score.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../styles/settings/colors/tokens';
+
+$badge-score-colors: (
```

Comment:
does it needs to be placed in a separate file?

Reply:
@kvas-damian I added it only for one reason, other "structures" with tokens have separated files f.e. `Button` , `IconButton`, `SurveyToggle` (in this PR it is not visible). I think it's good practice, main files are smaller.

Reply:
IMO, when such structure is used only in one file/component such separation makes it harder to fully understand the code - you have to switch between tabs.
Having that said, I have to proofs to confirm that. It's just my feeling, so I won't force you to change that

Reply:
I'm also not forcing my change. It's not logic change, so reverting it is a moment. I'm also not sure it's more clear, let it stay as it is and if it will give us any problems with code reading, we can back to previous setting.